content/en/docs/how-tos/cluster-claim.md

```diff
@@ -212,11 +253,27 @@ document.
 
 ## Troubleshooting Cluster Pools
 
+### Access Cluster Installation logs
 The cluster pools are maintained by Hive behind the scenes, so installation failures, cloud platform account
 misconfigurations and similar issues are not exposed to actual CI jobs: the jobs will simply never successfully claim a
-cluster if Hive fails to install them. Currently, cluster pool owners cannot access the logs or other troubleshooting
-information on the `hive` cluster. For now, work with DPTP (ping `@dptp-helpdesk` on #forum-testplatform Slack channel)
-to troubleshoot pools that never get ready clusters.
+cluster if Hive fails to install them.
+
+The installation logs can be found in the `hive` container logs with the following commands:
+
+```console
+# filter out the namespaces start with the cluster pool's name
+$ oc get ns | grep dptp-demo-cluster-pool
+dptp-demo-cluster-pool-pclkt              Active   16m
+
+# get the provision pod name
+$ oc get pod -n dptp-demo-cluster-pool-pclkt
+NAME                                                            READY   STATUS     RESTARTS   AGE
+dptp-demo-cluster-pool-pclkt-0-h5tml-provision-m2pp9   1/3     NotReady   0          18m
+
+# access the logs
+$ oc logs -n dptp-demo-cluster-pool-pclkt dptp-demo-cluster-pool-pclkt-0-h5tml-provision-m2pp9 -c hive
```

Comment:
Similarly... I would either present a script here or a one-liner with expansions `$()` - not a set of commands where copy-paste errors occur.

Reply:
See if it looks better now.